airbyte-ci/connectors/pipelines/pipelines/commands/groups/connectors.py

```diff
@@ -123,11 +123,11 @@ def get_selected_connectors_with_modified_files(
 )
 @click.option("--language", "languages", multiple=True, help="Filter connectors to test by language.", type=click.Choice(ConnectorLanguage))
 @click.option(
-    "--release-stage",
-    "release_stages",
+    "--support-level",
+    "support_levels",
     multiple=True,
-    help="Filter connectors to test by release stage.",
-    type=click.Choice(["alpha", "beta", "generally_available"]),
+    help="Filter connectors to test by support_level.",
+    type=click.Choice(["community", "certified"]),
```

Comment:
nit: I did made the extra mile to create an enum for releases stages, but I think it's worth it. (doing the same as ConnectorLanguage)